seatunnel-examples/seatunnel-flink-examples/src/main/resources/examples/fake_to_console.config

```diff
@@ -0,0 +1,55 @@
+##
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Please keep the license format consistent

Reply:
> Please keep the license format consistent

done